code/modules/food&drinks/drinks/drinks/drinkingglass.dm

```diff
@@ -94,7 +94,7 @@
 			if("goldschlager")
 				icon_state = "goldschlagerglass"
 				name = "glass of Goldschlager"
-				desc = "100 proof that teen girls will drink anything with gold in it."
```

Comment:
I thought this was pun on the usage of the word "proof" in drinks to denote percentage of alcohol. Goldschlager is traditionally between 87-107 proof so it's pretty accurate.